packages/styles/scss/theme/_foundation.scss

```diff
@@ -87,6 +80,45 @@
   --ilo-breakpoint-xxl: #{map-get($breakpoints-foundation, "xxl")};
   --ilo-border-radius: calc(#{px-to-rem(2)} * var(--ilo-scale));
 
+  /**
+    * Font-size
+    */
+  --ilo-font-size-2xsm: #{px-to-rem(10)};
+  --ilo-font-size-xsm: #{px-to-rem(12)};
+  --ilo-font-size-sm: #{px-to-rem(14)};
+  --ilo-font-size-md: #{px-to-rem(16)};
+  --ilo-font-size-lg: #{px-to-rem(18)};
+  --ilo-font-size-xlg: #{px-to-rem(24)};
+  --ilo-font-size-2xlg: #{px-to-rem(28)};
+  --ilo-font-size-3xlg: #{px-to-rem(32)};
+  --ilo-font-size-4xlg: #{px-to-rem(36)};
+  --ilo-font-size-5xlg: #{px-to-rem(44)};
```

Comment:
@GGKapanadze I have a dumb question because I don't remember what we decided here. The scale value for the foundations file is 1. so we're just multiplying all these values by 1?

Reply:
Yep, the initial value for the scale will be one but it will be exposed to anyone who wants to use the theme and can be easily scaled